game/static/game/css/board.css

```
@@ -1256,14 +1260,43 @@ body {
 
+.theme-option .theme-btn {
+    flex-shrink: 0;
+    margin: 0;
+    /* Lock the button sizes so they NEVER squish or change shape */
+    width: 18px !important; 
+    height: 18px !important;
+    border-width: 1.5px;
+}
+
+.theme-label {
+    font-size: clamp(6.5px, 2.2vw, 8px);    color: rgba(255, 255, 255, 0.75);
```

Stylelint (17.12.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)